### PR TITLE
Fix 2 bugs in Linux environment

### PR DESCRIPTION
**Pull request by mzch**
Fix 2 bugs in Linux environment

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,6 +69,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "firebase": "^7.11.0",
+    "postcss-cli": "^7.1.0",
```
Comment (or-else) [inline package.json]: Why is this a run-time dependence? Shouldn't it be in `"devDependencies"` section?

File: src/views/messages-view.jsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ import DocPreview from '../widgets/doc-preview.jsx';
 import ErrorPanel from '../widgets/error-panel.jsx';
 import GroupSubs from '../widgets/group-subs.jsx';
 import ImagePreview from '../widgets/image-preview.jsx';
-import Invitation from '../widgets/Invitation.jsx';
+import Invitation from '../widgets/invitation.jsx';
```
Comment (or-else) [inline src/views/messages-view.jsx]: That's indeed a bug. Good catch. Thanks.

File: package.json (hunk 1 above)
Comment (mzch) [reply package.json]: postcss-cli is needed in `npm run build`. Of course when running `npm run` separately, it's not needed.

Comment (or-else) [reply package.json]: Yes, I understand. But you placed it into `"dependencies"` section when it's needed in `"devDependencies"` section.  

```
  "dependencies": {
    "firebase": "^7.11.0", 
                                                 <<<--- NOT here
    "react": "^16.13.0",
    "react-dom": "^16.13.0",
    "react-intl": "^3.12.1",
    "tinode-sdk": "^0.16.4"
  },
  "devDependencies": {
    "@babel/core": "^7.8.7",
    "@babel/preset-env": "^7.8.7",
    "@babel/preset-react": "^7.8.3",
    "babel-loader": "^8.0.6",
    "babel-plugin-react-intl": "^5.1.18",
    "browserslist": "^4.9.1",
    "copy-webpack-plugin": "^5.1.1",
    "cssnano": "^4.1.10",
    "postcss": "^7.0.27",
    "postcss-cli": "^7.1.0",                        <<<< --- HERE
    "terser-webpack-plugin": "^2.3.5",
    "webpack": "^4.42.0",
    "webpack-cli": "^3.3.11"
  }
```

Comment (mzch) [reply package.json]: I see. :)

**Pull request merged**